### ISSUE TITLE
Umbraco.MultiNodeTreePicker issues

### ISSUE THREAD

**Issue by nvisage-gf** [KevinJump/uSync3.0#137]
Hi,

Having some issue with current release:

- MultiNodeTreePicker failing to export(deletes existing config from disk)
- Log file is showing error loading mapper for Umbraco.MultiNodeTreePicker (trying to load assembly Newtonsoft.Json, Version=10.0.0.0)
- uSyncCore.Config is modified on startup stripping all Mappings except Umbraco.MultiNodeTreePicker and media removed from MappingType).

Using Umbraco 7.5.14 and default Newtonsoft dependency (6.0.8)

**Comment by nvisage-gf** [comment]
Sample picker xml config:

```xml
  <DataType Name="Site Content MultiPicker" Key="a32d530e-cd25-4490-af96-cb24f2cb6c25" Id="Umbraco.MultiNodeTreePicker" DatabaseType="Nvarchar" Folder="Custom">
    <PreValues>
      <PreValue Id="36" Value="{&#xD;&#xA;  &quot;type&quot;: &quot;content&quot;,&#xD;&#xA;  &quot;query&quot;: &quot;$site/website&quot;&#xD;&#xA;}" Alias="startNode" />
      <PreValue Id="37" Value="home,contentPage,landingPage,newsListing,search,thankYou" Alias="filter" />
      <PreValue Id="38" Value="0" Alias="minNumber" />
      <PreValue Id="39" Value="8" Alias="maxNumber" />
      <PreValue Id="40" Value="1" Alias="showOpenButton" />
      <PreValue Id="41" Value="0" Alias="showEditButton" />
      <PreValue Id="42" Value="1" Alias="showPathOnHover" />
    </PreValues>
  </DataType>
```

[umbraco-log-extract.txt](https://github.com/KevinJump/uSync/files/1063559/umbraco-log-extract.txt)

uSync v3.3.4.740
uSync.Core v5.6.4.760

Will try downgrading uSync.Core to .740

**Comment by nvisage-gf** [comment]
asmspy output:

```
  Reference: Newtonsoft.Json
    Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed
  Source: NotFound
      10.0.0.0 by Jumoo.uSync.Core, Version=5.6.4.760, Culture=neutral, PublicKeyToken=null
```

**Comment by KevinJump** [comment]
yeah - i don't think 5.6.4.760 should have been released - that will have been targeted to umb 7.6 which is using newtonsoft 10.0 

I would down grade to .740 for now, I will go check why/what i did !

**Comment by KevinJump** [comment]
think i published the wrong package :( there is now a 5.6.4.740 version of core - and i have unlisted the 4.760 version, can you try that ?

**Comment by nvisage-gf** [comment]
Rolled back to 5.6.4.740 and issues resolved; thanks for quick turnaround!
